### DISCUSSION TITLE
Support multiple prompts attached to a generation.

### DISCUSSION THREAD

**Discussion by jansenderr** [langfuse/langfuse#2065] [Ideas]
### Describe the feature or potential improvement

For when I am using nested prompts / multiple prompts to create one system prompt I'd like to make sure ALL prompts are being counted on the generation.

Currently a generation takes a singular "prompt" parameter. This should be expanded to accept an array of prompts so tracking for all the prompts which make up the generation is accurate.

**Comment by marcklingen** [comment]
Hi Jansen, thanks for adding this. Makes a lot of sense!
